### DISCUSSION TITLE
phantom-camera0.73 BUG

### DISCUSSION THREAD

**Discussion by MRWWWWW1** [ramokz/phantom-camera#379] [Question / Help Wanted]
The 4.22 stable version I downloaded from spine officially crashed out as soon as I copied the plugin into the root directory with version 0.73

**Comment by ramokz** [comment]
Not sure how you experienced that, installs just fine for me in Godot 4.2.2.

What do you mean by "spine officially"?

**Comment by MRWWWWW1** [comment]
&nbsp;I downloaded version 4.22 from here(http://zh.esotericsoftware.com/spine-godot )&nbsp; .
When I unzip the camera to the root directory,He ducked back.




godotidle
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ramokz/phantom-camera"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年9月9日(星期一) 晚上7:15
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [ramokz/phantom-camera] phantom-camera0.73 BUG (Discussion #379)





 
Not sure how you experienced that, installs just fine for me in Godot 4.2.2.
 
What do you mean by "spine officially"?

**Reply by ramokz** [reply]
Sorry, I don't think it will be possible for me to support Godot releases outside Godot's official channels at this time.
Especially if there are no errors or anything to debug. For all intents and purposes, it could come down to how they compiled the engine, which would be my guess.

**Comment by MRWWWWW1** [comment]
It doesn't solve the problem, but thank you for your answer
&nbsp;


godotidle
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Marcus ***@***.***&gt;; 
发送时间: 2024年9月9日(星期一) 晚上8:17
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [ramokz/phantom-camera] phantom-camera0.73 BUG (Discussion #379)















抱歉，我认为目前我不可能在戈多官方渠道之外支持戈多发行作品。
特别是如果没有错误或需要调试的东西。总的来说，这可以归结为他们是如何编译引擎的，这是我的猜测。

-
直接回复这封邮件，在GitHub上查看，或取消订阅.
***@***.*** &gt;
